src/app/app.component.html

```diff
@@ -1,293 +1,104 @@
-              <span class="bb-mega-menu__item-description ng-star-inserted">Bill pay</span>
-              <bb-icon-ui name="toggle-down"
-                class="bb-mega-menu__expand-marker bb-mega-menu__expand-marker--vertical ng-star-inserted"><i
-                  aria-hidden="true" class="bb-icon bb-icon-toggle-down bb-icon--md"></i></bb-icon-ui>
-              <bb-icon-ui name="toggle-down"
-                class="bb-mega-menu__expand-marker bb-mega-menu__expand-marker--horizontal ng-star-inserted"><i
-                  aria-hidden="true" class="bb-icon bb-icon-chevron-right bb-icon--md"></i></bb-icon-ui>
-            </a>
-          </div>
-          <div class="bb-mega-menu__submenu bb-mega-menu__submenu--cols1 ng-star-inserted" role="region" tabindex="-1"
-            aria-labelledby="bb-menu-header-button-billpay" hidden="">
-            <div class="bb-mega-menu__submenu-items"><a class="bb-mega-menu__submenu-header" href="#" role="button">
-                <bb-icon-ui name="toggle-down" class="bb-mega-menu__submenu--close" name="angle-left" size="lg"><i
-                    aria-hidden="true" class="bb-icon bb-icon-angle-left bb-icon--lg"></i>
-                </bb-icon-ui>
-                <div class="bb-mega-menu__submenu--text"> Main Menu</div>
-              </a>
-              <div class="bb-mega-menu__column bb-mega-menu__column--links ng-star-inserted">
-                <a bbrouterlinkactive="bb-mega-menu__submenu-link--active"
-                  class="bb-mega-menu__submenu-link ng-star-inserted" href="#/billpay/pay-bill">
-                  <div class="bb-mega-menu__submenu-item-label ng-star-inserted">
-                    <div class="bb-mega-menu__submenu-title">Pay a bill</div>
-                  </div>
-                </a><a bbrouterlinkactive="bb-mega-menu__submenu-link--active"
-                  class="bb-mega-menu__submenu-link ng-star-inserted" href="#/billpay/pending-bills">
-
-                  <div class="bb-mega-menu__submenu-item-label ng-star-inserted">
-                    <div class="bb-mega-menu__submenu-title">Pending payments</div>
-                  </div>
-                </a><a bbrouterlinkactive="bb-mega-menu__submenu-link--active"
-                  class="bb-mega-menu__submenu-link ng-star-inserted" href="#/billpay/history-bills">
+<div class="bb-layout bb-layout--horizontal-nav"
+[ngClass]="{
+  'bb-layout--nav-expanded': layoutService.navigationExpanded$ | async,   
+  'bb-layout--high-resolution-view': layoutService.isHighResolutionView$ | async
+  }"
+>
+<!-- Topbar Area -->
+<nav class="bb-layout__topbar">
+  <!-- Skip to content -->
+  <button
+    i18n-aria-label="
+      Move focus to beginning of content|Value for aria-label saying that this button will move the focus to the
+      beginning of the content
+    "
+    aria-label="skip to content"
+    class="bb-layout__skip-to-content"
+    (click)="$any($event).view?.window?.document?.querySelector('[role=\'main\']')?.focus()"
```

Comment:
Can you move this logic to ts file, this looks hacky to me

Reply:
I am following the best practices provided by Web-UI team. This is the community documentation that I followed https://designsystem.backbase.com/v2/web-components/horizontal and this is supposed to be the Backbase way. Of course we can make it better and contribute.

Reply:
fixed